sdk/storage/README.md

```diff
@@ -147,131 +152,173 @@ The Azure Storage SDK for JavaScript provides low-level and high-level APIs. Tak
 
 - The high-level APIs provide convenience abstractions such as uploading a large stream to a block blob (using multiple PutBlock requests).
 
+## Examples
+
 ## Code Samples
 
 ```javascript
 const {
   BlobServiceClient,
   newPipeline,
-  SharedKeyCredential,
-  AnonymousCredential,
-  TokenCredential
+  SharedKeyCredential
 } = require("@azure/storage-blob");
 
 async function main() {
   // Enter your storage account name and shared key
-  const account = "account";
-  const accountKey = "accountkey";
+  const account = "";
+  const accountKey = "";
 
   // Use SharedKeyCredential with storage account and account key
   const sharedKeyCredential = new SharedKeyCredential(account, accountKey);
 
-  // Use TokenCredential with OAuth token
-  const tokenCredential = new TokenCredential("token");
-  tokenCredential.token = "renewedToken"; // Renew the token by updating token field of token credential
-
-  // Use AnonymousCredential when url already includes a SAS signature
-  const anonymousCredential = new AnonymousCredential();
-
   // Use sharedKeyCredential, tokenCredential or anonymousCredential to create a pipeline
   const pipeline = newPipeline(sharedKeyCredential);
 
```

Comment:
Seems some samples code are still demostating pipeline, and others not. Can we have an alignment?

Reply:
#4000 is logged to track this work.

---
In reply to: [296154082](https://github.com/Azure/azure-sdk-for-js/pull/3744#discussion_r296154082) [](ancestors = 296154082)